crates/viewer/re_renderer/src/video/player.rs

```diff
@@ -452,6 +455,15 @@ impl VideoPlayer {
         else if requested.gop_idx < last_requested.gop_idx {
             self.reset(video_description)?;
         }
+        // Previously signaled the end of the video, but encountering frames that are newer than the last enqueued.
+        else if self.signaled_end_of_video
```

Comment:
note this is two changes:
* moved up so it will be hit even if below `else if` is also true (which then may or may not reset)
* check last enqueued against last sample in the description